NTumbleBit/PuzzleSolver/SolverClientSession.cs

```diff
@@ -118,353 +108,324 @@ public State GetInternalState()
                         blinds[realI++] = real.BlindFactor;
                     }
                 }
-				state.FakeSolutions = fakeSolutions;
-				state.BlindFactors = blinds;
-				state.Commitments = commitments;
-				state.Puzzles = puzzles;
-			}
-			return state;
-		}
-
-		public new class State : EscrowInitiator.State
-		{
-			public PuzzleValue Puzzle
-			{
-				get; set;
-			}
-			public SolverClientStates Status
-			{
-				get; set;
-			}
-			public PuzzleSolution PuzzleSolution
-			{
-				get; set;
-			}
-			public PuzzleValue[] Puzzles
-			{
-				get; set;
-			}
-			public ServerCommitment[] Commitments
-			{
-				get; set;
-			}
-			public int[] FakeIndexes
-			{
-				get; set;
-			}
-			public PuzzleSolution[] FakeSolutions
-			{
-				get;
-				set;
-			}
-			public BlindFactor[] BlindFactors
-			{
-				get;
-				set;
-			}
-			public ScriptCoin OfferCoin
-			{
-				get;
-				set;
-			}
-		}
-
-
-		public SolverParameters Parameters
-		{
-			get
-			{
-				return _Parameters;
-			}
-		}
-
-		public RsaPubKey ServerKey
-		{
-			get
-			{
-				return _Parameters.ServerKey;
-			}
-		}
-
-		public SolverClientStates Status
-		{
-			get
-			{
-				return InternalState.Status;
-			}
-		}
-
-		public override void ConfigureEscrowedCoin(uint160 channelId, ScriptCoin escrowedCoin, Key escrowKey, Script redeemDestination)
-		{
-			AssertState(SolverClientStates.WaitingEscrow);
-			base.ConfigureEscrowedCoin(channelId, escrowedCoin, escrowKey, redeemDestination);
```

Comment:
I think this line got deleted by accident? Or am I missing something?

Reply:
Hi just checked the method is there - looks like this file was heavily formatted.

![image](https://user-images.githubusercontent.com/16000683/41872786-e4ab4bfc-78ba-11e8-8eb0-d74f94946cc8.png)